app/models/card.rb

```diff
@@ -27,6 +27,21 @@ def check_translation(user_translation)
     end
   end
 
+  def pending_cards_notification
```

Comment:
Assignment Branch Condition size for pending_cards_notification is too high. [15.43/15]